api/Performance.json

```diff
@@ -915,6 +916,7 @@
       "navigation": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/navigation",
+          "spec_url": "https://w3c.github.io/navigation-timing/#dom-performance-navigation",
```

Comment:
Do we really want this one? It just tells that this is obsolete and kept only for backward compatibility.

Reply:
I think it'd still be useful to have a spec URL defined, which developers can refer to if they so choose.  They're marked as deprecated here on BCD, so that should get the point across that they shouldn't be used in projects any longer!

Reply:
I'm ok with this. Should this be documented in some meta-docs?